operator/check_operator.go

```diff
@@ -92,6 +93,7 @@ func (c *operatorCheck) resolve(ctx context.Context) error {
 		DockerConfig:            c.dockerConfigFilePath,
 		Channel:                 c.operatorChannel,
 		Kubeconfig:              c.kubeconfig,
+		CSVTimeout:              c.csvTimeout,
```

Comment:
Unless I'm misreading, this value is 0 if the caller didn't include the option. We guard the user from this same problem by setting a default for the scorecard wait time when the check is instantiated, and probably need to do that for this option as well.

Reply:
Default file was created for this and setting default in root cmd like other defaults.